css-layout-classes/styles.css

```diff
@@ -0,0 +1,43 @@
+.container {
+  max-width: 800px;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
```

Comment:
I would avoid using display flex on the container itself. That is typically for the rows.

Reply:
Awesome! Thanks for the feedback!